[IL][HDX-10701] quickstart doc (#75)
hydrolix/mcp-hydrolix: README.md

Reviewer: now may be a good time to check these paths -- they've almost definitely changed, given the churn we've seen from anthropic

Reviewer: now is (sorry) the right time to update this as well (and all other examples) to use:
`uvx --python 3.13 --refresh-package mcp-hydrolix mcp-hydrolix`

This offers a couple benefits:
- `uvx` makes a fully isolated environment, regardless of whether it's run from inside a project -- this is desirable as "how to run our packaged app" (and someone who's for some reason wanting to run mcp-hydrolix in the context of a different uv project they're developing will already know that they can `uv run` stuff from their project)
- `--refresh-package mcp-hydrolix` ensures the latest is pulled (we've already had an internal user miss out on a feature change because they had an old cached/`tool install`-ed version)

An MCP server for Hydrolix.

## Quickstart

> Get up and running in a few minutes. This guide covers Claude Desktop (the most common setup) and Claude Code.

Reviewer: ```suggestion
Get up and running in a few minutes. This section covers Claude Desktop and Claude Code.
```
I think the assertion here is unnecessary, and possibly not even true

Reviewer: ("section" instead of "guide" since this could be read as "the README won't cover the more general setup instructions so don't bother looking")

### Step 1 — Prerequisites

Before you begin, make sure you have:

- **Hydrolix credentials** — your cluster hostname plus either a username/password or a service account token. If you don't have these, ask your Hydrolix administrator.
- **Claude Desktop** — download from [claude.ai/download](https://claude.ai/download).

### Step 2 — Install the MCP server

Choose the method that matches your setup:

**Option A: Using uv (recommended)**

[uv](https://docs.astral.sh/uv/) manages Python automatically. If you don't have it, install it first:

macOS / Linux:
```bash
curl -LsSf https://astral.sh/uv/install.sh | sh
```

Windows (PowerShell):
```powershell
powershell -ExecutionPolicy ByPass -c "irm https://astral.sh/uv/install.ps1 | iex"
```

Then install the MCP server:
```bash
uv tool install mcp-hydrolix

> ```

Reviewer: We should skip this -- better to run directly with `uv run --with` or `uvx`. The quick start should be consistent with the in-depth instructions later in the guide

Reviewer: See my third comment

Author: Good catch. Updated docs to use uvx

**Option B: Using pip**

Requires Python 3.13+. If you need to install Python, download it from [python.org](https://www.python.org/downloads/).

```bash
pip install mcp-hydrolix
```

### Step 3 — Configure Claude Desktop

1. Open the Claude Desktop configuration file:
   - **macOS:** `~/Library/Application Support/Claude/claude_desktop_config.json`
   - **Windows:** `%APPDATA%\Claude\claude_desktop_config.json`
   - **Linux:** `~/.config/Claude/claude_desktop_config.json`

2. Add the following entry to the `"mcpServers"` object (create the file with this content if it doesn't exist yet):

```json
{
  "mcpServers": {
    "mcp-hydrolix": {

>       "command": "mcp-hydrolix",

Reviewer 2: Might be  using `uvx` as the command instead of `mcp-hydrolix` — it's more reliable since Claude Desktop may not find binaries in the  PATH:
```json
 "command": "uvx",
 "args": ["mcp-hydrolix"]
 ```

Author: Good recommendation. I've updated the doc to use uvx

> **Using a service account token instead of username/password?** See [Authentication](#authentication).

> **Command not found?** Claude Desktop launches without your shell's PATH, so it may not locate the `mcp-hydrolix` binary even if installation succeeded. Find the full path by running the appropriate command in your terminal and use it as the `"command"` value in the config:
> - **macOS / Linux:** `which mcp-hydrolix`
> - **Windows:** `where.exe mcp-hydrolix`
>
> If `which`/`where.exe` returns nothing, your install location isn't on your PATH. For uv, try `uv tool dir --bin` to find where tools are installed. For pip, run `python3 -c "import sysconfig; print(sysconfig.get_path('scripts'))"` (macOS/Linux) or `python -c "import sysconfig; print(sysconfig.get_path('scripts'))"` (Windows) — this prints the directory where pip placed the executable. Run it using the same Python interpreter you used to pip install (e.g., activate your virtual environment first if you used one).

### Step 4 — Restart Claude Desktop

Restart the app to apply the configuration.

> > **Windows users:** Make sure to fully quit Claude via the system tray icon before restarting.

Reviewer: Does this not apply to macs as well? I thought it "closes" to the Menu Bar

Author: It does. I was assuming mac users would know to quit their apps instead of just closing it but I should make it clear.

Replace `<your-hydrolix-hostname>`, `<your-username>`, and `<your-password>` with your actual credentials.

> > **Using Option B (pip)?** Use `"command": "mcp-hydrolix"` with no `"args"` field instead.

Reviewer: For a quickstart, we can omit the pip instructions. It just needs to target the #1 most high-impact / high-value deployment setup. We can leave pip adaptations to the more detailed configuration docs
